src/model/block/ISetGroupMembershipBlockConfig.ts

```diff
@@ -1,23 +1,23 @@
 import {IBlockConfig} from '../..'
 
-export interface ISetGroupMembershipBlockConfig extends IBlockConfig {
-  /**
-   * An identifier for the group that membership will be set within.
-   */
+export interface IMembershipGroup {
```

Comment:
I'd love to keep `GroupMembership` word, because we might need to import TS interface in flow-builder or other repo. Using the same word would help devs to access this interface easily.
How about `IGroupMembership` or `IGroupMembershipOption`?

- [ ] try to keep the `GroupMembership` word